### PR TITLE
trivial: Add new category type for video display

### PR DESCRIPTION
**Pull request by gportay**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [x] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-engine.c [hunk 1]
```diff
@@ -3786,6 +3786,8 @@ fu_common_device_category_to_name(const gchar *cat)
 		return "Mouse";
 	if (g_strcmp0(cat, "X-Keyboard") == 0)
 		return "Keyboard";
+	if (g_strcmp0(cat, "X-VideoDisplay") == 0)
+		return "Video Display";
```
Comment (hughsie) [inline src/fu-engine.c:3790]: Maybe just Display? at least to my ears `Video Display` seems a bit odd to hear in 2022.

Comment (gportay) [reply src/fu-engine.c:3790]: It was to match the icon name.

okay for the string.

Comment (gportay) [reply src/fu-engine.c]: done.

**Review by hughsie — APPROVED** [review]

**Comment by hughsie** [comment]
I've also added the category to the LVFS too.

**Pull request merged**